test/services/host_inventory_api_test.rb

```diff
@@ -30,25 +30,8 @@ class HostInventoryApiTest < ActiveSupport::TestCase
     assert_equal @host.id, @api.host_already_in_inventory(@host.id)['id']
   end
 
-  test 'create_host_in_inventory' do
-    response = OpenStruct.new(
-      body: { data: [{ host: @inventory_host }] }.to_json,
-      success?: true
-    )
-
-    @connection.expects(:post).returns(response)
-    assert_equal @host.id, @api.create_host_in_inventory['id']
-  end
-
   test 'inventory_host for host already in inventory' do
     @api.expects(:host_already_in_inventory).returns(@host)
-    @api.expects(:create_host_in_inventory).never
-    assert_equal @host, @api.inventory_host
-  end
-
-  test 'inventory_host for host not already in inventory' do
```

Comment:
We still need this test, but updated to assert that nil is returned when the host isn't found.